framework/src/play/vfs/VirtualFile.java

```diff
@@ -1,30 +1,25 @@
 package play.vfs;
 
-import org.apache.commons.io.IOUtils;
-import play.Play;
 import play.exceptions.UnexpectedException;
 
 import javax.annotation.CheckReturnValue;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import java.io.File;
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.Writer;
 import java.net.URI;
-import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map.Entry;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 
-import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.apache.commons.io.FileUtils.readFileToByteArray;
-import static org.apache.commons.io.FileUtils.readFileToString;
 
+/**
+ * @deprecated This class is not needed anymore. 
+ * It was used in Play! framework to load resources from local folder "modules" which was a custom dependency management mechanism.
+ * RePlay uses standard Maven/Gradle dependencies. Folder "modules" doesn't exist anymore.
+ * 
+ * So we use plain old `java.util.File` instead.
```

Comment:
There is a typo: `java.io.File`.

```suggestion
 * So we use plain old {@link java.io.File} instead.
```
While at it: link the class instead just formatting it

Reply:
thanks, fixed the link.